server/src/main/java/org/opensearch/telemetry/tracing/SpanBuilder.java

```diff
@@ -128,6 +138,20 @@ private static Attributes buildSpanAttributes(String action, Transport.Connectio
         return attributes;
     }
 
+    private static Attributes buildSpanAttributes(String nodeId, BulkShardRequest bulkShardRequest) {
+        Attributes attributes = buildSpanAttributes(nodeId, bulkShardRequest.shardId());
+        attributes.addAttribute(AttributeNames.NUM_BULK_ITEMS, bulkShardRequest.items().length);
```

Comment:
Can we add RefreshPolicy as well (if it is not `NONE`) as an attribute?

Reply:
Added in latest revision